tracer/build/_build/PrepareRelease/SetAllVersions.cs

```diff
@@ -283,7 +288,8 @@ public override void Run()
                 // Four-part AssemblyVersion update
                 SynchronizeVersion(
                     "src/Datadog.Trace/TracerConstants.cs",
-                    FourPartVersionReplace);
+                    // upgrading four part, then three part *seems* safe
+                    text => ThreePartSemVerReplace(FourPartVersionReplace(text)));
```

Comment:
Instead of replacing the assembly-wide constant here, could we have the `RcmRequest` class break the four-part version into a cached three-part version?

Reply:
Ignore my comment, your approach uses a new SemVer version field 👍🏼